templates/contact-us.html

```diff
@@ -21,23 +21,114 @@
-        <p class="u-sv1">Our <a href="https://www.ubuntu.com/support">support team</a> team offers a range of services from enterprise support to fully managed infrastructure and applications.</p>
+        <p class="u-sv1">Our <a href="https://www.ubuntu.com/support">support team</a> offers a range of services from enterprise support to fully managed infrastructure and applications.</p>
         <p class="p-heading--three">Trademark and other licenses</p>
-        <p class="u-sv1">Publishing or distributing Ubuntu in any form other than our standard install media and packages requires permission. To license the Ubuntu trademark or any other rights, please <a href="https://www.ubuntu.com/legal/terms-and-policies/contact-us">submit an enquiry</a>.</p>
+        <p class="u-sv1">Publishing or distributing Ubuntu in any form other than our standard install media and packages
+          requires permission. To license the Ubuntu trademark or any other rights, please <a
+            href="https://www.ubuntu.com/legal/terms-and-policies/contact-us">submit an enquiry</a>.</p>
         <p class="p-heading--three">Press and analyst enquiries</p>
         <p class="u-sv1">Please contact <a href="mailto:pr@canonical.com">pr@canonical.com</a>.</p>
         <p class="p-heading--three">Data Privacy enquiries</p>
-        <p class="u-sv1">For information about data privacy, please read our <a href="https://www.ubuntu.com/legal/data-privacy">Privacy Policy</a> or <a href="https://ubuntu.com/legal/data-privacy/enquiry">submit an enquiry</a>. A member of our data privacy team will be in touch with you shortly.</p>
+        <p class="u-sv1">For information about data privacy, please read our <a
+            href="https://www.ubuntu.com/legal/data-privacy">Privacy Policy</a> or <a
+            href="https://ubuntu.com/legal/data-privacy/enquiry">submit an enquiry</a>. A member of our data privacy team
+          will be in touch with you shortly.</p>
         <p class="p-heading--three">Legal enquiries</p>
-        <p>Please note that we cannot provide you with legal advice. For any other queries, please contact us on <a href="mailto:legal@canonical.com">legal@canonical.com</a>.</p>
+        <p>Please note that we cannot provide you with legal advice. For any other queries, please contact us on <a
+            href="mailto:legal@canonical.com">legal@canonical.com</a>.</p>
       </div>
     </div>
   </section>
-{% endblock %}
+  <section class="p-strip--light is-shallow" style="padding-bottom: 6rem;">
+    <div class="u-fixed-width">
+      <p class="p-heading--three">
+        <a href="#" class="p-reveal-link js-location-toggle-button">Where we are</a>
+      </p>
+    </div>
+    <div class="js-location-container">
+      <div class="u-fixed-width">
+        <hr style="margin: 2rem 0;" />
+      </div>
+      <div class="row">
+        <div class="col-4 p-card" style="border-style: none;">
+          <p class="p-card__title">Canonical Group Limited</p>
```

Comment:
Would need @waynecrosby's thoughts on the look of it, but I think these should be `h3` headers, possibly with `p-heading--five` applied (which maintains the font size, but increases the font weight):
```suggestion
          <h3 class="p-card__title p-heading--five">Canonical Group Limited</h3>
```

Reply:
It doesn't look the same if i use `p-heading--five`